### PR TITLE
[DEV-3693] Add Prayer Preview Card

### PR DESCRIPTION
**Pull request by richarddubay**
### What does this PR do, or why is it needed?
This PR adds a prayer preview card and displays it when you tap on either the church, campus, or community tab of the prayer menu.
Screenshot 2019-04-22 13 30 54

### How do I test this PR?
1. Pull this branch
2. Start up the simulator
3. Tap on the prayer menu icon
4. Tap on the menu tabs. The prayer preview card should appear when you tap on the church, campus, or community tabs. This is static content and will not change per tab for MVP. 

---

> I am affirming this is my _best_ work ([Ecclesiastes 9:10](https://www.bible.com/bible/97/ECC.9.10.MSG))

## TODO

- [x] PR has a relevant title that will be understandable in a public changelog (ie...non developers)
- [x] It is a portion of  [DEV-3693](https://newspring.atlassian.net/browse/DEV-3693)
- [x] No new warnings in tests, in storybook, and in-app
- [x] Upload GIF(s) of iOS and Android if applicable
- [x] Set two relevant reviewers

## REVIEW

**Manual QA**

- [x] Manual QA on iOS and ensure it looks/behaves as expected
- [ ] Manual QA on Android and ensure it looks/behaves as expected

**Code Review: Questions to consider**

- [ ] Read through the "Files changed" tab _very carefully_
- [ ] Edge cases: what assumptions are made about input?
- [ ] What kind of tests could be written?
- [ ] How might we make this easier for someone else to understand?
- [ ] Could the code be simpler?
- [ ] Will the code be easy to modify in the future?
- [ ] What's one part of these changes that makes you excited to merge it?

---

> The purpose of PR Review is to _improve the quality of the software._

### CODE REVIEW

**Comment by richarddubay** [comment]
This still needs a tweak to get it to live at the bottom of the viewport but I wanted to move this along. I can add that tweak to my list of polish items unless someone knows how to make it happen.

**Review by cdanwards — COMMENTED** [review]
@richarddubay I ran tests and the code looks good. On IOS this works as intended, but on Android there's still the issue of the tabs not displaying content so I can't speak to Android working as intended.

File: packages/newspringchurchapp/src/prayer/PrayerPreviewCard/PrayerPreviewCard.stories.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    uri: 'https://fillmurray.com/400/600',
+  },
+  name: 'Bill',
+  overlayColor: ['#FFF', '#FFF'],
+  prayer:
+    'I’m alright. Nobody worry ’bout me. Why you got to gimme a fight? Can’t you just let it be?',
+  source: 'Anderson',
+};
+
+storiesOf('PrayerPreviewCard', module).add('Example', () => (
```
Comment (redreceipt) [inline packages/newspringchurchapp/src/prayer/PrayerPreviewCard/PrayerPreviewCard.stories.js]:
```suggestion
storiesOf('prayer/PrayerPreviewCard', module).add('Example', () => (
```

Comment (redreceipt) [reply packages/newspringchurchapp/src/prayer/PrayerPreviewCard/PrayerPreviewCard.stories.js]: I would throw all your stories inside the prayer category

File: packages/newspringchurchapp/src/prayer/PrayerPreviewCard/index.js [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+  marginHorizontal: 0,
+  marginVertical: 0,
+}))(Card);
+
+const StyledCardContent = styled(() => ({
+  alignItems: 'center',
+}))(CardContent);
+
+const StyledBodyText = styled(({ theme }) => ({
+  marginTop: theme.sizing.baseUnit * 1.5,
```
Comment (redreceipt) [inline packages/newspringchurchapp/src/prayer/PrayerPreviewCard/index.js:61]: probably should add `margin` to a container around these according to our other convos instead of the text itself.

File: packages/newspringchurchapp/src/ui/PrayerMenu/index.js [hunk 3]
```diff
@@ -28,6 +26,27 @@ const loadingStateObject = {
   },
 };
```
Comment (redreceipt) [inline packages/newspringchurchapp/src/ui/PrayerMenu/index.js]:
```suggestion
// TODO: remove once this is pulling data
```

File: packages/newspringchurchapp/src/ui/PrayerMenu/index.js [hunk 4]
```diff
@@ -28,6 +26,27 @@ const loadingStateObject = {
...
+    }}
+    name={'Bill'}
+    overlayColor={['#FFF', '#FFF']}
+    prayer={
+      'I’m alright. Nobody worry ’bout me. Why you got to gimme a fight? Can’t you just let it be?'
+    }
+    source={'Anderson'}
+  />
+);
+
```
Comment (redreceipt) [inline packages/newspringchurchapp/src/ui/PrayerMenu/index.js]:
```suggestion
// TODO: remove once this is pulling data
```

File: packages/newspringchurchapp/src/ui/PrayerMenu/index.js [hunk 5]
```diff
@@ -28,6 +26,27 @@ const loadingStateObject = {
...
+const StyledPrayerPreviewCardComponent = styled({
+  position: 'absolute',
+  bottom: 0,
+  left: 0,
+  right: 0,
+})(PrayerPreviewCardComponent);
+
 const prayerMenuData = [
```
Comment (redreceipt) [inline packages/newspringchurchapp/src/ui/PrayerMenu/index.js:51]:
```suggestion
// TODO: this needs to be dynamic at some point
const prayerMenuData = [
```

**Review by redreceipt — DISMISSED** [review]

**Comment by IsaacHardy** [comment]
@redreceipt We already have a tag for "waiting on changes". No need to create a new one.

**Comment by richarddubay** [comment]
> @redreceipt We already have a tag for "waiting on changes". No need to create a new one.

I think they actually both existed. Feel free to delete one!

File: packages/newspringchurchapp/src/prayer/PrayerPreviewCard/index.js (hunk 2 above)
Comment (richarddubay) [reply packages/newspringchurchapp/src/prayer/PrayerPreviewCard/index.js:61]: This is a good call. Moved the margin to a container.

**Review by cdanwards — APPROVED** [review]
Tested and looks good! Great job @richarddubay !

**Review by IsaacHardy — APPROVED** [review]
Looks good.
